src/api/app/services/notification_service/web_channel.rb

```diff
@@ -9,7 +9,8 @@ class WebChannel
       'Report' => OutdatedNotificationsFinder::Report,
       'Decision' => OutdatedNotificationsFinder::Decision,
       'Appeal' => OutdatedNotificationsFinder::Appeal,
-      'WorkflowRun' => OutdatedNotificationsFinder::WorkflowRun
+      'WorkflowRun' => OutdatedNotificationsFinder::WorkflowRun,
+      'Group' => OutdatedNotificationsFinder::Group
```

Comment:
I think we can get rid of this table. See https://trello.com/c/krml3hbT/184-remove-the-allowedfinders-from-notificationservice

Reply:
I have to double-check and take a decision.

Reply:
I mean, not now. I wrote a card to do it later...

Reply:
To start with, the finder was wrong because it used `@parameters['notifiable_id']` instead of `@parameters[:notifiable_id]`. There are other wrong finders, so I created this [Trello card](https://trello.com/c/UwjKoROV/186-revisit-outdatednotficationsfinder).

The goal of this finder is to avoid repeating the same type of notification many times. If someone adds and removes user U from group G many times, we keep only the last addition and removal. So I believe it makes sense to keep the functionality.

Please check the latest commit to see the fix.